### ISSUE TITLE
Display bug on iPhone 5 regarding QR scan and LAPP explorer

### ISSUE THREAD

**Issue by Pythoniasm** [BlueWallet/BlueWallet#423]
This bug occurs since around one month, I can't nail it to a commit nor date. The LAPP browser already existed and fully and correctly displayed all LAPPs at the beginning, though.

## Bug
When viewing the LAPP Explorer or using the QR scan feature through smartphone camera, around 5/6th of the screen is blocked by a plain white ``div`` area.

With this bug, only a small 1 cm wide area remains on top to position the camera view when scanning the QR code or display around one LAPP item, but not more on top of the display. The Rest is hidden beneath the blank white area.

## System
iPhone 5 with iOS 10 (last supported build)

**Comment by Overtorment** [comment]
Attach a screenshot pls. Would not hurt. Looks like an issue with marketplace html coding.
cc @ncoelho

**Comment by Pythoniasm** [comment]
Below are the requested screenshots:

### QR-Scan
![qr_no_exif](https://user-images.githubusercontent.com/26092465/54500027-e0bc7280-4918-11e9-8cd2-478b7eba9131.jpg)

### LAPP
![lapp_no_exif](https://user-images.githubusercontent.com/26092465/54500029-e5812680-4918-11e9-88ba-3761747590fc.jpg)

**Comment by Pythoniasm** [comment]
@Overtorment I hope you celebrate your next commit - it will be the number 1000!

**Comment by Overtorment** [comment]
I had the same issue on my old iPad, with old iOS. Then I updated iOS, that solved it

**Comment by Pythoniasm** [comment]
While I do understand that code support for old phone models is impossible, I would like to point out three things:

1. I do not have the possibility to update my iOS, due to the fact that iPhone 5 uses a 32bit architecture which is not supported by iOS versions > v10. Updating iOS means buying a new device. Since there is an outright death of "small" display smartphones on the market right now, I see people forced to run their old model until a hardware issue.
2. BlueWallet worked for me flawlessly, until recently. Including the time when I used it heavily during my first PR on this repository. This bug is tied to recent changes in the code.

Hence, I expect that other OS versions on Android or even iOS might have a similar bug and an investigation of the source is not completely out of scope. However, this is obviously no priority for the majority with state of the art hardware.

**Comment by Overtorment** [comment]
This is because we updated ReactNative, and latest version doesn't play
nice with old iOS.
